content/chapter-03/Summary.md

```diff
@@ -1 +1,7 @@
-# Chapter 03 - Summary
\ No newline at end of file
+# Chapter 03 - Summary
+
+* There are multiple metrics we can obtain from the system we are using
+* System metrics on the on the public cloud are more relevant than on-premise as we need to configure them directly
+* In the public cloud we need to understand better the performance of our application as it will lead to less infrastructure usage and less money to pay for it
+* There are multiple problems an application could face but to make them first-class citizens on our design is becoming not optional
```

Comment:
There are multiple challenges an application could face and to make them more reliable our design patterns are becoming crucial for a long-term success

Reply:
Done